src/pages/vault/downloads/index.tsx

```diff
@@ -0,0 +1,16 @@
+import BaseNewLayout from 'layouts/base-new'
+import { ReactElement } from 'react'
+
+const VaultDownloadsPage = (): ReactElement => {
```

Comment:
Stubbed this in simply so there's something to link to in the top nav (as has been done for Waypoint).